runtime/src/bank.rs

```diff
@@ -1823,10 +1823,11 @@ impl Bank {
         // from Stakes<Delegation> by reading the full account state from
         // accounts-db. Note that it is crucial that these accounts are loaded
         // at the right slot and match precisely with serialized Delegations.
+
```

Comment:
was this extra blank line intentional?

Reply:
It was because it's two distinct paragraphs. But both are about the same operation (Stakes::new), so I've joined them with "//\n" now